packages-exp/auth-exp/src/core/persistence/index.ts

```diff
@@ -31,10 +31,16 @@ export type PersistenceValue = PersistedBlob | string;
 
 export const STORAGE_AVAILABLE_KEY = '__sak';
 
+export interface StorageEventListener {
+  (value: PersistenceValue | null): void;
+}
+
 export interface Persistence {
```

Comment:
All of these should have `_` prefixed names since they're publicly visible but "package private" but this PR is already pretty big so that's just something to fix later

Reply:
easy to do in vscode, done